cdc/processor/pipeline/sink.go

```diff
@@ -137,7 +137,7 @@ func (n *sinkNode) flushSink(ctx context.Context, resolved model.ResolvedTs) (er
 		// 2. tableActor resolvedTs >= processor resolvedTs >= global resolvedTs >= barrierTs
 		redoTs := n.redoManager.GetMinResolvedTs()
 		if redoTs < currentBarrierTs {
-			log.Debug("redoTs should not less than current barrierTs",
+			log.Fatal("redoTs should not less than current barrierTs",
```

Comment:
```suggestion
			log.Panic("redoTs should not less than current barrierTs",
```

Reply:
Whether `schedulerV2` might violate this constraint?

Reply:
I'm not sure. I think it won't break the consitraint because
1. barrierTs is global min **redo-flushed** table resolved timestamp
2. redoManager.minResolvedTs can only regress when a table is moved in it

@overvenus PTAL thanks.

Reply:
The log level has been reverted to `Warn` because it can really happen.